src/AnonymizationStep/AnonymizedResultsTable.tsx

```diff
@@ -97,22 +98,23 @@ function makeColumnData(title: string, dataIndex: string, type: ColumnType, rend
 
 const AGG_COLUMN_TYPE = 'real';
 
-const mapColumn = (mode: DisplayMode, bucketColumns: BucketColumn[]) => (column: AnonymizedResultColumn, i: number) => {
-  if (column.type === 'aggregate') {
-    switch (mode) {
-      case 'anonymized':
-        return [makeColumnData(column.name, i + '_anon', AGG_COLUMN_TYPE, renderValue)];
-      case 'combined':
-        return [
-          makeColumnData(column.name + ' (anonymized)', i + '_anon', AGG_COLUMN_TYPE, renderLowCountValue),
-          makeColumnData(column.name + ' (original)', i + '_real', AGG_COLUMN_TYPE, renderValue),
-          makeColumnData('Distortion', i + '_diff', 'real', renderRelativeNoiseValue),
-        ];
+const mapColumn =
+  (mode: DisplayMode, bucketColumns: BucketColumn[]) => (column: AnonymizedResultColumn, columnIdx: number) => {
+    if (column.type === 'aggregate') {
+      switch (mode) {
+        case 'anonymized':
+          return [makeColumnData(column.name, columnIdx + '_anon', AGG_COLUMN_TYPE, renderValue)];
+        case 'combined':
+          return [
+            makeColumnData(column.name + ' (anonymized)', columnIdx + '_anon', AGG_COLUMN_TYPE, renderLowCountValue),
+            makeColumnData(column.name + ' (original)', columnIdx + '_real', AGG_COLUMN_TYPE, renderValue),
+            makeColumnData('Distortion', columnIdx + '_diff', 'real', renderRelativeNoiseValue),
+          ];
+      }
     }
-  }
 
-  return [makeColumnData(column.name, i.toString(), column.type, buildCellRenderer(column, bucketColumns))];
-};
+    return [makeColumnData(column.name, columnIdx.toString(), column.type, buildCellRenderer(column, bucketColumns))];
+  };
```

Comment:
this is another "indent-induced-diff" - the only change I've made here is to rename `i` to `columnIdx`, which changed the formatting with indent.